README.md

```diff
@@ -34,6 +34,9 @@
   5. [Objects and Data Structures](#objects-and-data-structures)
      * [Use object encapsulation](#use-object-encapsulation)
      * [Make objects have private/protected members](#make-objects-have-privateprotected-members)
+  6. [Error handling](#error-handling)
+     * [Use Exceptions Rather Than Return Codes](#use-exceptions-rather-than-return-codes)
+     * [Don't Return Null](#dont-return-null)
   6. [Classes](#classes)
```

Comment:
Even if Markdown replace correctly number in ordered list, we should update the other numbers for lisibility ?